docs/standard/commandline/customize-help.md

```diff
@@ -0,0 +1,115 @@
+---
+title: How to customize help in System.CommandLine
+description: "Learn how to customize help in apps that are built with the System.Commandline library."
+ms.date: 01/25/2022
+no-loc: [System.CommandLine]
+helpviewer_keywords:
+  - "command line interface"
+  - "command line"
+  - "System.CommandLine"
+ms.topic: how-to
+---
+# How to customize help in apps that are built with the System.Commandline library
+
+You can customize help for a specific command, option, or argument, and you can add or replace whole help sections.
+
+The examples in this article work with the following command-line application:
+
+:::code language="csharp" source="snippets/customize-help/csharp/Program.cs" id="original" :::
+
+When help is requested, the following output is produced:
+
+```output
+Description:
+  Read a file
+
+Usage:
+  scl [options]
+
+Options:
+  --file <file>                                               The file to print out. [default: scl.runtimeconfig.json]
+  --light-mode                                                Determines whether the background color will be black or
+                                                              white
+  --color                                                     Specifies the foreground color of console output
+  <Black|Blue|Cyan|DarkBlue|DarkCyan|DarkGray|DarkGreen|Dark  [default: White]
```

Comment:
Do we have an issue for finding a solution here that is less ugly (a System.CommandLine, not docs question)

Reply:
Help customization allows people to change this output.